.github/workflows/cypress_e2e_tests.yml

```diff
@@ -0,0 +1,81 @@
+name: Cypress E2E Tests
+on:
+  schedule:
+    - cron: '0 1 * * *'  # every day at 1 am
+
+jobs:
+  cypress-run:
+    environment: Test
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Cypress Dependency
+        working-directory: ./integrationTests      
+        env:
+          NODE_ENV: development
+          OKTA_DOMAIN: ${{ secrets.OKTA_DOMAIN }}
+          OKTA_API_KEY: ${{ secrets.OKTA_API_KEY }}          
+        run: |
+          sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+          sudo chmod +x /usr/local/bin/docker-compose
+          sudo ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose
+          npm cache clean --force
+          npm install
+      - name: Cypress Dependency Web
+        working-directory: ./web      
+        env:
+          NODE_ENV: development
+          OKTA_DOMAIN: ${{ secrets.OKTA_DOMAIN }}
+          OKTA_API_KEY: ${{ secrets.OKTA_API_KEY }}          
+        run: |
+          npm install
+      - name: Cypress Dependency API
+        working-directory: ./api
+        env:
+          NODE_ENV: development
+          OKTA_DOMAIN: ${{ secrets.OKTA_DOMAIN }}
+          OKTA_API_KEY: ${{ secrets.OKTA_API_KEY }}          
+        run: |
+          npm install

```

Comment:
This whole section seems unnecessary bc this won't run in this environment. Same for the web one below.  The dependencies are needed inside the container where they should be installed when the container is built.

Reply:
I misunderstood what is going on.  This makes sense now.